cli/tests/integrations/test_cluster.py

```diff
@@ -43,3 +44,21 @@ def test_rename():
 
     # rename back to original name
     assert_command(['dcos', 'cluster', 'rename', new_name, name])
+
+
+def test_setup_noninteractive():
+    """
+    Run "dcos cluster setup" command as non-interactive with a 30 sec timeout.
+    This makes sure the process doesn't prompt for input forever (DCOS-15590).
+    """
+
+    try:
+        returncode, stdout, _ = exec_command(
+            ['dcos', 'cluster', 'setup',
```

Comment:
All arguments at the same level should either be on the same line or split onto individual lines, e.g. not
```
returncode, stdout, _ = exec_command(
    ['dcos', 'cluster', 'setup',
        'https://dcos.snakeoil.mesosphere.com'],
    timeout=30, stdin=subprocess.DEVNULL)
```
but rather
```
returncode, stdout, _ = exec_command(
    ['dcos', 'cluster', 'setup', 'https://dcos.snakeoil.mesosphere.com'],
    timeout=30,
    stdin=subprocess.DEVNULL)
```
or
```
returncode, stdout, _ = exec_command(
    ['dcos',
     'cluster',
     'setup',
     'https://dcos.snakeoil.mesosphere.com'],
    timeout=30,
    stdin=subprocess.DEVNULL)
```

It makes it much easier to look at and understand without haveing to spend alot of time reading the code.